[2024-06-22T07:38:58.789+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-22T07:38:58.799+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_v2.second_task manual__2024-06-22T07:34:55.796357+00:00 [queued]>
[2024-06-22T07:38:58.803+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_v2.second_task manual__2024-06-22T07:34:55.796357+00:00 [queued]>
[2024-06-22T07:38:58.803+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-06-22T07:38:58.808+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-06-22 07:34:55.796357+00:00
[2024-06-22T07:38:58.810+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1954) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-22T07:38:58.811+0000] {standard_task_runner.py:63} INFO - Started process 1955 to run task
[2024-06-22T07:38:58.811+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag_v2', 'second_task', 'manual__2024-06-22T07:34:55.796357+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp1syl0ald']
[2024-06-22T07:38:58.812+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask second_task
[2024-06-22T07:38:58.832+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag_v2.second_task manual__2024-06-22T07:34:55.796357+00:00 [running]> on host 65f8df8d6172
[2024-06-22T07:38:58.865+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gnosia93' AIRFLOW_CTX_DAG_ID='first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T07:34:55.796357+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-22T07:34:55.796357+00:00'
[2024-06-22T07:38:58.866+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-22T07:38:58.872+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-22T07:38:58.872+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo I am second task, .. gnosia93']
[2024-06-22T07:38:58.876+0000] {subprocess.py:86} INFO - Output:
[2024-06-22T07:38:58.877+0000] {subprocess.py:93} INFO - I am second task, .. gnosia93
[2024-06-22T07:38:58.877+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-22T07:38:58.878+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-22T07:38:58.889+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag_v2, task_id=second_task, run_id=manual__2024-06-22T07:34:55.796357+00:00, execution_date=20240622T073455, start_date=20240622T073858, end_date=20240622T073858
[2024-06-22T07:38:58.909+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-22T07:38:58.918+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-22T07:38:58.919+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
